src/oncall/ui/static/js/incalendar.js

```diff
@@ -1245,6 +1245,14 @@
               })
             )
           )
+          .append(
```

Comment:
Looks like you can add a note, but it doesn't actually appear in the UI when viewing event details or editing.